moco_control/src/moco_hw_interface.cpp

```diff
@@ -95,9 +100,9 @@ bool MocoHWInterface::init() {
     }  // end for each joint
 
     registerInterface(&joint_state_interface_);     // From RobotHW base class.
-    registerInterface(&position_joint_interface_);  // From RobotHW base class.
     registerInterface(&velocity_joint_interface_);  // From RobotHW base class.
```

Comment:
Does the `position_joint_interface_` need to go away? Alternatively asked: Should the `velocity_joint_interface_` also not be registered?

I'm a little unclear if the joint limits still function this way.

Reply:
I'm also unclear on that, and can't test it without hardware

Reply:
Make an issue and we'll look at it later?

Reply:
I reverted this line and made an issue